src/vlog/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.utils import timezone
+from django_extensions.db.models import TimeStampedModel
+
+from contrib.timescale.fields import TimescaleDateTimeField
+
+
+class Vlog(TimeStampedModel):
+    time = TimescaleDateTimeField(interval="1 week", default=timezone.now)
+
+    # VRI stands for VerkeersRegelInstallatie. One VRI contains all
+    # the installations in one intersection.
+    vri_id = models.IntegerField()
+
+    # Max 255 types can exist
+    message_type = models.PositiveSmallIntegerField()
+
+    # The message itself.
+    message = models.CharField(max_length=255)
```

Comment:
is 255 lang genoeg voor de message?

Reply:
ja de langste die ik tot nog toe gezien heb is 24 chars